src/app/shop/[category]/[productId]/_components/TabContents/Review/ReviewImageList.tsx

```diff
@@ -1,23 +1,83 @@
+'use client';
+
+import { Modal } from '@/components';
 import RenderImages from '@/components/ReviewItem/RenderImages';
+import { ChevronIcon } from '@/public/index';
 import type { ReviewImage } from '@/types/ProductReviewTypes';
 import classNames from 'classnames/bind';
+import { useState } from 'react';
 import styles from './ReviewImageList.module.scss';
+import ReviewImageModal from './ReviewImageModal';
 
 const cn = classNames.bind(styles);
 interface ReviewImageListProps {
+  productId: number;
   reviewImgs: ReviewImage[];
 }
 
-export default function ReviewImageList({ reviewImgs }: ReviewImageListProps) {
+const SECTION_SIZE = 7;
+
+export default function ReviewImageList({ productId, reviewImgs }: ReviewImageListProps) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [reviewId, setReviewId] = useState<number>(0);
+  const [currentSectionIndex, setCurrentSectionIndex] = useState(0);
+
+  const isPrevDisabled = currentSectionIndex === 0;
+  const isNextDisabled = currentSectionIndex === Math.floor(reviewImgs.length / SECTION_SIZE) - 1;
```

Comment:
1 안 빼도 될 것 같아요. reviewImgs가 빈 배열일 때 -1이라서 제대로 표시가 안되고 있어요!
```suggestion
  const isNextDisabled = currentSectionIndex === Math.floor(reviewImgs.length / SECTION_SIZE);
```